run-spread

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# check dependencies
+if ! snap list snapcraft &>/dev/null; then
+    echo "snapcraft is not installed"
+    exit 1
+fi
+
+# Clean the snaps created in previous runs
+touch test-build
+rm -rf built-snap
+mkdir built-snap
+rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
+
+# Build snapd snap
+snapcraft --use-lxd
+for snap in built-snap/snapd_1337.*.snap; do
+    mv "${snap}" "${snap}.keep"
+done
+
+# Run spread
+#shellcheck disable=SC2068
+spread $@
```

Comment:
```suggestion
spread "$@"
```

Drop the shellcheck exception, this was a real bug.

Reply:
sure, thanks